YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -688,6 +701,12 @@ bool Repository::ask_and_set_git_config_entry(const std::string& config_entry, c
     return true;
 }
 
+bool Repository::is_file_tracked(const std::string& path)
+{
+    bool is_tracked = !repo_.get_untracked_objects_in_path(path).empty();
```

Comment:
nit: just return !repo_. ...

Reply:
fixed in 1d1e98c73c4b5c6eec04e07bac56ccb00b47b515